src/Microsoft.AspNetCore.Build/scripts/KoreBuild.sh

```diff
@@ -125,6 +124,18 @@ ensure_msbuild
 
 KOREBUILD_TARGETS_ROOT="$KOREBUILD_ROOT/src/Microsoft.AspNetCore.Build/targets"
 
+MSBUILD_RSP=$(mktemp -t koremsbuild)
+
+cat > $MSBUILD_RSP <<ENDMSBUILDARGS
```

Comment:
Do you need rsp files in xplat? I thought they were only needed on Windows because cmd had odd argument size restrictions.